src/schema.rs

```diff
@@ -601,6 +641,13 @@ impl WebsiteData {
     }
 }
 
+#[derive(serde_derive::Deserialize, Debug)]
+#[serde(deny_unknown_fields)]
+pub(crate) struct MemberRole {
```

Comment:
`MemberRole` in contrast with `DiscordRole`. Not sure if this is a great name. `Role` did not seem adequate.

https://github.com/rust-lang/team/blob/57e50ecf446550dc0ef48f8e48bcc3b8abada10e/src/schema.rs#L465-L468